Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ however, it has to be formatted properly to pass verification tests.
 
 ### Changed
 
+- Changed `TrackedPoseDriver` to use properties of type `InputActionProperty` rather than `InputAction` to allow more
```

Comment:
Looks like the end of the sentence is missing?

Reply:
Fixed, also found there was a merge error that wasnt building and fixed it.  That got me thinking that I am not sure whether this change has had an coverage from the quality team so I added @jackpr-unity to the PR.